AirlineDetails/DetailsViewModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  DetailsViewModel.swift
+//  AirlinesApplication
+//
+//  Created by admin user on 20/07/2024.
+//
+
+import Foundation
+import UIKit
+
+class DetailsViewModel{
```

Comment:
we can convert it to struct not class 
- struct is a value type so it's performance better than class